core/src/main/res/values/strings.xml

```diff
@@ -790,56 +790,56 @@
     <string name="port_label">Port</string>
     <string name="optional_hint">(Optional)</string>
     <string name="proxy_test_label">Test</string>
-    <string name="proxy_checking">Checking&#8230;</string>
-    <string name="proxy_test_successful">Test successful</string>
+    <string name="proxy_checking">Checking…</string>
+    <string name="proxy_test_successful">Done</string>
```

Comment:
Done and successful are different things

Reply:
It was able to test, and/or the test ran?